test/e2e/search_test.go

```diff
@@ -10,9 +12,14 @@ import (
 
 var _ = Describe("Podman search", func() {
 	var (
-		tempdir    string
-		err        error
-		podmanTest PodmanTest
+		tempdir         string
+		err             error
+		podmanTest      PodmanTest
+		regFileContents = `[registries.search]
+    registries = []
```

Comment:
Fix the format

Reply:
How would you suggest formatting it? I based it off of libpod/runtime_img_test.go:13 but these globals are indented in a tab because of the Describe function, so I'm not sure what would look/work the best

Reply:
I'd move the large string out of the `var` block and into a separate `const` where it doesn't mess with the indentation

Reply:
Something like this? Or would it be better if the contents were tabbed in one more time?
```
var _ = Describe("Podman search", func() {
	var (
		tempdir    string
		err        error
		podmanTest PodmanTest
	)
    const regFileContents = `
    [registries.search]
    registries = []

    [registries.insecure]
    registries = ['localhost:5000']`

```

Reply:
Yeah, that looks fine to me